src/client/players.py

```diff
@@ -45,19 +45,25 @@ def isFriend(self, id):
         '''
         Convenience function for other modules to inquire about a user's friendliness.
         '''
+        if id in self._logins:
+            id = self._logins[id].id
```

Comment:
The coexisting lookup scheme is the player name, so id might be a user name string.

The code is still transitioning from using names everywhere into using IDs instead, but until that's done I feel like the isFriend and isFoe methods should support the old way of doing things.